### PR TITLE
Convert XML to UDL

### PR DESCRIPTION
**Pull request by jakcpto**
In purpose of easy development and debugging

### CODE REVIEW

File: MDX2JSON/Utils.cls [hunk 1]
```diff
@@ -151,37 +116,27 @@ Automatic processing of MDX query and outputting resulting JSON.<br>
...
+ClassMethod WriteDrillthroughJSON(pMDX As %String) As %Status
+{
 	set RS = ..GetResultSet(pMDX,.st)
 	return:$$$ISERR(st) st
 	set st = ..ExecuteResultSet(RS)
 	return:$$$ISERR(st) st
 	
 	set SQL = RS.%GetListingSQL()
-	do ##class(%ZEN.Auxiliary.altJSONSQLProvider).%WriteJSONFromSQL(,SQL,,$$$MaxCacheInt)
+	do ##class(%ZEN.Auxiliary.jsonSQLProvider).%WriteJSONFromSQL(,SQL,,$$$MaxCacheInt)
```
Comment (eduard93) [inline MDX2JSON/Utils.cls:131]: Why is this fix not in? https://github.com/intersystems-ru/Cache-MDX2JSON/commit/78f29b7183277d6a3f18dd070e578d4d582fb5ac

**Comment by jakcpto** [comment]
Sorry, just by mistake.

ср, 21 дек. 2022 г., 13:11 Eduard ***@***.***>:

File: MDX2JSON/Utils.cls (hunk 1 above)
Comment (jakcpto) [reply MDX2JSON/Utils.cls:131]: Just by mistake. Sorry (

**Review by jakcpto — COMMENTED** [review]
Fixed altJSONSQLProvider

**Pull request merged**